conda.recipe/meta.yaml

```diff
@@ -8,7 +8,7 @@ package:
 
 source:
   git_url: ../
-  # git_rev: {{ 'v' + setup_data['version'] }}  # always use tagged version
+  git_rev: {{ 'v' + setup_data['version'] }}  # always use tagged version
```

Comment:
The reason I had commented that out is because you can't test building a package if you haven't tagged yet.